src/index.html

```diff
@@ -9,6 +9,7 @@
     />
   </head>
   <body>
+    <div id="tree"></div>
```

Comment:
The HTML file is linking to a SCSS file, which is not directly usable by browsers. Ensure that the SCSS is compiled to CSS and link to the resulting CSS file instead.